Controllers/CartController.cs

```diff
@@ -37,7 +38,7 @@ public CartController(BangazonContext ctx)
             // TODO: This is a placeholder value. These two lines should be removed after the User Accounts dropdown works
             if (SessionHelper.ActiveUser == null)
```

Comment:
Let's kill this reference to SessionHelper.ActiveUser. Don't need this class anymore since we've done the whole singleton thing